pkgs/applications/misc/visidata/default.nix

```diff
@@ -20,19 +20,22 @@
 , wcwidth
 , zstandard
 , setuptools
+, importlib-metadata
 , git
 , withPcap ? true, dpkt, dnslib
 , withXclip ? stdenv.isLinux, xclip
+, testers
+, visidata
```

Comment:
Is it better to do something with `let self = … ; in self`? This approach seems fragile w.r.t. overrides.

Reply:
I was following https://nixos.org/manual/nixpkgs/unstable/#tester-testVersion . Looking at other instances of `testers.testVersion` , I do not see that `let self` construct.

Also, if you want to override this package, you would probably not care about this simplistic test, which just checks for the version output. This is mostly ( i think ) intended to aid automatic updates.

Reply:
Well true that if there are never more tests, this approach is fine…